internal/own/codeowners/proto/find_owners_test.go

```diff
@@ -23,6 +23,22 @@ func TestFileOwnersMatch(t *testing.T) {
 				"/prefix/filename",
 			},
 		},
+		{
+			pattern: "*.md",
+			paths: []string{
+				"/README.md",
```

Comment:
because we talked about it earlier today

```suggestion
				"/README.md",
				"/README.md.md",
```